config/initializers/health_check.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+HealthCheck.setup do |config|
+  # uri prefix (no leading slash)
+  config.uri = "healthcheck"
+
+  # Text output upon success
+  config.success = "success for following checks: ['database', 'migrations', 'email']"
```

Comment:
So I get what database and migrations checks do (or can do, more or less). 

What does email check do?

Reply:
I don't think we should be checking email as we are using GOV Notify service